### DISCUSSION TITLE
How do I properly update PHP and all extensions to version 8.1.x only?

### DISCUSSION THREAD

**Discussion by droplinxuser** [oerdnj/deb.sury.org#1904] [Q&A] [answered]
I want to avoid upgrading to PHP 8.2.x for now, as not all hosted scripts are compatible with it.
Thanks!

**Comment by oerdnj** [comment, marked as answer]
There's this lovely [FAQ](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions) that you forgot to read.

**Comment by droplinxuser** [comment]
Sorry for being a newbie in this field.
Is there a way to prevent a downtime, since this is a live server with live websites?

Based on the guide, I can only fix it by removing the generic package and installing the specific version package, which includes:
```
apt remove php-fpm
apt install php8.1-fpm
```

**Comment by droplinxuser** [comment]
Solved now with few seconds downtime.

`sudo apt remove php* -y; sudo apt install -y php8.1-fpm`
